.github/workflows/publish_deploy_image.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
 
   deploy-to-openshift-test:
     if: github.event.ref == 'refs/heads/main'
-    needs: [build-and-push-image]
+    needs: [tests, build-and-push-image]
```

Comment:
While this additional pipeline improvement isn't about Sysdig, I agree the test env should have the same stage requirements that dev env has

Reply:
should I revert this and open a new ticket for it?

Reply:
No I think this is a great change. Usually would be a separate PR, yes, but leave it in :)